metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py

```diff
@@ -924,18 +956,22 @@ def get_workunits(self) -> Iterable[MetadataWorkUnit]:
                 # Convert PowerBi Dashboard and child entities to Datahub work unit to ingest into Datahub
                 workunits = self.mapper.to_datahub_work_units(dashboard, workspace)
                 for workunit in workunits:
-                    # Add workunit to report
-                    self.reporter.report_workunit(workunit)
                     # Return workunit to Datahub Ingestion framework
                     yield workunit
 
-            if self.source_config.extract_reports:
-                for report in self.powerbi_client.get_reports(workspace=workspace):
-                    for work_unit in self.mapper.report_to_datahub_work_units(
-                        report, workspace
-                    ):
-                        self.reporter.report_workunit(work_unit)
-                        yield work_unit
+            for report in workspace.reports:
```

Comment:
Did we remove the extract_reports config? Is this safe?

Reply:
yes, it is safe, the flag check is moved into powerbi_api.py::_fill_regular_metadata_detail